config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
+    'last 2 Chrome versions',
+    'last 2 Firefox versions',
+    'last 1 Safari versions',
+    'last 2 Edge versions'
+  ]
+};
```

Comment:
No, this was just a judgment call on my part as part of the upgrade. I decided to increase the support from 1 version to two for the major browsers and to drop IE9 in exchange for Edge (which is EOL and a weird default).